src/schemas/author_schema.py

```diff
@@ -12,8 +12,8 @@ class AuthorSchema(BaseModel):
     """
 
     id: Optional[int] = None
-    first_name: str = Field(title="First name of Author ")
-    last_name: str = Field(title="Last name of Author ")
+    first_name: constr(strip_whitespace=True, min_length=1, max_length=32)
+    last_name: constr(strip_whitespace=True, min_length=1, max_length=32)
     birth_date: datetime = Field(title="Date of Birth For The Author")
```

Comment:
I see the type for birth and death date is datetime, instead of just date. Don't you think birth_date and death_date should be just dates? Also does this mean the user has to provide a full time datetime string for it to get accepted?

Reply:
Dates provided in format like `2023-04-26T00:00:00` are accepted. We didn't have any clear requirements regarding time so I guess if we are following the convention of using only the date I can change this.

Reply:
Updated! now simple dates are accepted like `2023-04-26`.

Reply:
Yes, this seems more correct as such dates should not contain time information